curl-and-grep/curl-and-grep.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: curl-and-grep
+spec:
+  inputs:
+    params:
+    - name: url
+      description: URL to be curl'ed.
+    - name: text
+      description: text to grep
+      default: "HTTP"
+    - name: buffer
+      description: file to save output
+      default: '/workspace/buffer.txt'
+  steps:
+  - name: grep
+    image: plerionio/curl
```

Comment:
This is alpine + curl - however do you know the image maintainer / if the image is maintained?

I wonder if we should add to the catalog repo machinery to include custom docker images and build them on a regular basis. This way we would be in control of the images we include in Tasks. 
For images that are well maintained like alpine, python, ubuntu or so it's fine to get them from dockerhub.

Reply:
Agreed, I need to find more trusted image.

Reply:
@afrittoli It seems like before we start to create such custom tasks, we really need to think about the trusted source of any kind of tools. For example, in case of curl, it's really difficult to find the trusted image. So, I'm with you on the catalog for custom images under the Tekton team control.